Platform/ElkhartlakeBoardPkg/CfgData/CfgData_Ext_IotgCrb.dlt

```diff
@@ -26,6 +26,7 @@ GPIO_CFG_DATA.GpioPinConfig1_GPP_D13.GPIOSkip_GPP_D13 | 0
 # Enable to test TCC mode & tuning
 # FEATURES_CFG_DATA.Features.Tcc    | 0x1
 FEATURES_CFG_DATA.Features.LowPowerIdle      | 0x1
+FEATURES_CFG_DATA.Features.NewGpioSchemeEnable      | 0x1

```

Comment:
Do you know if this change is compatible to all OS ?   EX: Does every POR OS move to the new GPIO scheme ?